### ISSUE TITLE
Create and implement a plan for testing the 10-10EZ Mailing Address

### ISSUE THREAD

**Issue by ala-yna** [department-of-veterans-affairs/va.gov-team#17541]
## Issue Description
- Given that we're adding two address fields to the 10-10EZ, how can we test that work and demo it with the stakeholder in a way that prevents accidental production deploy and eases the burden on both the product team and the stakeholders?
---
## Tasks
- [ ] _What work is necessary for this story to be completed?_

## Acceptance Criteria
- [ ] _What will be created or happen as a result of this story?_

**Comment by LinnJS** [comment]
Possible options of duplicating `permanentAddress`  if `mailingAddress` does not exist, question being where should we do this work.

1) Can do on pre-fill if no `mailingAddress` auto-fill `permanentAddress` into that field
- Don't want to due this due to inconveniencing the user of having duplicate fields if they change mind from profile data

2) Can do on transforming formData to schema data on FE before submitting to BE
- Most likely the case we are going to go with since most of the flips should be happing on FE already anyways
- Kevin and I also feel strongly that the `vet-json-schmea` should reflect business needs of `enrollment service`

3) Do this work on the BE before submitting to `enrollment service`

**Comment by kevinmirc** [comment]
_Justin & Kevin - Feature Flip discussion - 2020-12-16 11:36 PM_

The front end form will have "address is same" boolean prop. Before sending to API (in the transformer) will add mailing if the flag is true, and remove the flag. The backend will expect two addresses.

In order to deploy, we can make the mailing address optional first. Then followup PR to make it required.

The last question standing is how the enrollment system requires this address. After my (Kevin's) research tickets, and meeting w/ Josh, we should have an understanding of how/when to send the second address with a feature flip (separate URLs versions?).

We can add a shim to the backend that allows us to send data to enrollment when the feature flag is on/off. Then remove it when it's fully deployed, required on the schema, and the front end will always be sending it.

We should have this same conversation on Friday to outline the feature toggle (release plan) in detail, given the extra context we'll have by then.

**Comment by kevinmirc** [comment]
Updates
- 10-10EZ is able to accept additional properties not (yet) defined on the schema
- ES can accept one or two addresses, it does not block when receiving one address at this time

Given the above updates, we can deploy the backend without any schema updates or coordination with the front-end. We just allow it to accept one or two addresses (for when the flipper is disabled or enabled respectively).

Once all code is merged we can toggle the flipper on and the form will start collecting two addresses and they will flow through to ES.
